src/index.html

```diff
@@ -3,12 +3,72 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#apple"
+              class=" nav__item is-active"
+            >
+            APPLE
+            </a>
+          </li>
+
+          <li>
+            <a href="#samsung" class="nav__item">Samsung</a>
+          </li>
+
+          <li>
+            <a href="#smartphones" class="nav__item">Smartphones</a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops&computers"
+              data-qa="hover"
+              class="nav__item"
+            >
+            Laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#gadgets" class="nav__item">Gadgets</a>
+          </li>
+          </label>
+
+          <li>
+            <a href="#tablets" class="nav__item">Tablets</a>
+          </li>
+
+          <li>
+            <a href="#photo" class="nav__item">Photo</a>
+          </li>
+
+          <li>
+            <a href="#video" class="nav__item">Video</a>
+          </li>
+          
+        </ul>
+      </nav>
+
+    </header>
```

Comment:
між батьківським і дочірнім елементами не потрібен відступ
```suggestion
      </nav>
    </header>
```